client/dist/styles/questionsAnswers.css

```diff
@@ -339,18 +300,121 @@ h3.addQuestionBtn {
 .helpulQuestion.display {
   padding-right: 3em;
 }
-
 .bottomButtons {
   display: flex;
 }
-h3.moreAnsweredBtn {
+.moreAnsweredBtn {
   display: flex;
   margin-right: 2.5em;
   justify-content: start;
 }
-.answerImage.container {
+/*addAnswer component CSS*/
+.closeAnswer{
   display: flex;
-  flex-direction: row;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin: 0;
+  position:absolute;
+  top: 0vh;
+  left: 0vw;
+  height: 5vh;
+  width: 5vh;
+  border-radius: 50em;
+  background-color: lightsteelblue;
+  z-index: 1;
+}
+.closeAnswer:hover{
+  color: darkred;
+  cursor: pointer;
+}
+.answerFormPhotoThumbnail{
+  display: flex;
+  justify-content: center;
+}
+.addAnswerForm{
+  display: flex;
+  flex-direction: column;
+  margin: 0;
+  bottom: 25vh;
+}
+.currentQuestion {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 5vh;
+  margin: 0;
+}
+textarea.addAnswerFormText {
+ display: flex;
+ height: 15vh;
+ resize:none;
+ overflow-y: scroll;
+ justify-content: start;
+}
+.addAnswerFormNickNameInput{
+  width: 20vw;
+  overflow-x: auto;
+}
+.addAnswerFormEmail {
+  width: 20vw;
+  overflow-x: auto;
+}
+.addAnswerFormUploadPhotoTitle {
+  display: flex;
+  flex-direction: column;
+  margin-bottom: .25vh;
+}
+.answerFormAsterisk{
+  margin-bottom:.25vh;
+  width: 20vw;
+  overflow-x: auto;
+}
+.answerFormAsterisk.answer:after, .answerFormAsterisk.nickName:after, .answerFormAsterisk.email:after{
+  display: fixed;
+  content: ' *';
```

Comment:
I want to check that the space before the asterisk is intentional- I imagine it is, just want to make sure